pom.xml

```diff
@@ -3431,6 +3433,26 @@
 
     </profile>
 
+    <profile>
+      <id>jdk-17</id>
```

Comment:
Can we make this active by default on Java 17? Does it cause any conflicts with other profiles currently?
```
      <activation>
        <jdk>[17,)</jdk>
      </activation>
```

Reply:
I don't think that helps - this just controls whether it's active when _built_ with JDK 17. We're not building with 17 anytime soon